src/Controller/SettingsController.php

```diff
@@ -198,7 +198,7 @@ public function addFolderAction(Request $request): JsonResponse
             }
 
             if (is_writable(dirname($file))) {
-                File::mkdir($file);
+                $filesystem->mkdir($file);
```

Comment:
Please remove unused `use Pimcore\File;` import.

Reply:
Done in [60fb349](https://github.com/pimcore/file-explorer-bundle/pull/8/commits/60fb349172f56d2ee5a62274bea5ff20596199d9)